actions/user_actions.test.js

```diff
@@ -595,10 +583,4 @@ describe('Actions.User', () => {
         expect(UserActions.queue.onEmpty).toHaveBeenCalled();
         expect(UserActions.queue.add).toHaveBeenCalled();
     });
-
```

Comment:
Line 333 & 100 in this file, we no longer need `ExperimentalChannelSidebarOrganization`